demos/text_detection_demo/cpp/src/cnn.cpp

```diff
@@ -64,18 +64,21 @@ Cnn::Cnn(
     // Configuring input and output
     ov::preprocess::PrePostProcessor ppp(m_model);
 
-    // we'd like to pass input image (NCWH, u8) to model input
+    // we'd like to pass input image (NHWC, u8) to model input
     // and let OpenVINO do necessary conversions
+
     ppp.input().tensor()
         .set_element_type(ov::element::u8)
-        .set_layout({ "NHWC" })
-        .set_spatial_dynamic_shape();
+        .set_layout({ "NHWC" });
 
-    ppp.input().preprocess().convert_layout(input_layout);
```

Comment:
why removed? will this work with IR v10?

Reply:
layout conversion step is added when you set layout to tensor:
```
   ppp.input().tensor()
        .set_element_type(ov::element::u8)
        .set_layout({ "NHWC" });
```
We can see it in log:
```
[ INFO ] Preprocessor configuration:
[ INFO ] Input "Placeholder":
    User's input tensor: {1,768,1280,3}, [N,H,W,C], u8
    Model's expected tensor: {1,3,768,1280}, [N,C,H,W], f32
    Implicit pre-processing steps (2):
      convert type (f32): ({1,768,1280,3}, [N,H,W,C], u8) -> ({1,768,1280,3}, [N,H,W,C], f32)
      convert layout [N,C,H,W]: ({1,768,1280,3}, [N,H,W,C], f32) -> ({1,3,768,1280}, [N,C,H,W], f32)
```
no need to set it in preprocess.

Reply:
that was to make that obvious, but ok